From 82218a42eb25dde521dc20c20878973bf856dd03 Mon Sep 17 00:00:00 2001
From: Zhichao Yu <zhichao.yu@rock-chips.com>
Date: Sat, 9 May 2020 14:03:54 +0800
Subject: [PATCH] ARM: dts: rockchip: unlimited logging to /dev/kmsg for
 rv1126-evb-ddr3-v10-tb

printk.devkmsg={on,off,ratelimit}
	Control writing to /dev/kmsg.
	on - unlimited logging to /dev/kmsg from userspace
	off - logging to /dev/kmsg disabled
	ratelimit - ratelimit the logging
	Default: ratelimit

Signed-off-by: Zhichao Yu <zhichao.yu@rock-chips.com>
Change-Id: I69f5f4804990247b35e9aea3106e1536b8dc3be7
---
 arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts
index c835016ae571..7a6d64844bda 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts
@@ -10,7 +10,7 @@
 	compatible = "rockchip,rv1126-evb-ddr3-v10-tb", "rockchip,rv1126";
 
 	chosen {
-		bootargs = "loglevel=0 initcall_nr_threads=0 initcall_debug=0 root=/dev/rd0 console=ttyFIQ0 snd_aloop.index=7";
+		bootargs = "loglevel=0 initcall_nr_threads=0 initcall_debug=0 printk.devkmsg=on root=/dev/rd0 console=ttyFIQ0 snd_aloop.index=7";
 	};
 
 	memory {
-- 
2.35.3

